utils/rank.js

```diff
@@ -33,8 +33,18 @@ const getBounds = (ranks, index) => {
 	return ranks.slice(index - 1, index + 1);
 };
 
-export default (ranks, index, count = 1) => {
+export const findRank = (ranks, index, count = 1) => {
 	const [above, below] = getBounds(ranks, index);
 	const [result] = mudder.base36.mudder(above, below, count);
 	return result;
 };
+
+export const findRankInRankedList = (rankedList, index) =>
+	findRank(
+		rankedList.map((s) => s.rank),
+		index,
+	);
+
+export const sortByRank = (array) => {
+	return array.concat().sort((a, b) => (a.rank || '').localeCompare(b.rank || ''));
+};
```

Comment:
Nit, do we need the block / `return` here? Looks like in the util above we use the compact arrow function